cmd/main-binary/main.go

```diff
@@ -126,6 +126,12 @@ func main() {
 			},
 		}
 	}
+	if enableENoExecEventControllers {
+		leaderID = fmt.Sprintf("enoexecevent-controllers-%s", leaderID)
+		cacheOpts.DefaultNamespaces = map[string]cache.Config{
+			"openshift-multiarch-tuning-operator": {},
```

Comment:
```suggestion
			utils.Namespace(): {},
```
Can this be changed to avoid hardcoding the name?